src/service/ampdoc-impl.js

```diff
@@ -300,6 +303,14 @@ export class AmpDoc {
   contains(node) {
     return this.getRootNode().contains(node);
   }
+
+  /**
+   * Binding of 3p analytics vendors' responses, used amp-ad-exit
+   * @return {!Object<string, !Object<string, Object<string,string>>>}
+   */
+  getIframeTransportResponses() {
```

Comment:
So...shall we leave this in ampdoc-impl, or create a new service, or something else?